frontend/src/App.vue

```diff
@@ -59,6 +59,7 @@ export default {
 <!-- these styles must be global -->
 <!-- eslint-disable-next-line vue-scoped-css/enforce-style-type -->
 <style lang="scss">
+@import 'src/scss/tailwind';
```

Comment:
Why is this part of this PR?

Reply:
I want to move all tailwind like classes to a file we can delete afterwards the migration to tailwind.